src/test/index.spec.ts

```diff
@@ -1955,4 +2053,144 @@ test.suite('ts-node', (test) => {
       });
     }
   });
+
+  test.suite('top level await', (test) => {
+    const compilerOptions = {
+      target: semver.gte(ts.version, '3.0.1')
+        ? 'es2018'
+        : // TODO should this be kept if tla is not supported on TS <3.8.0?
```

Comment:
I think that we could default to `transpileOnly=true` when TS is <3.8.0, so tests still run here (and target would be removed)

Reply:
I'd rather not risk the inevitable bug report from someone confused as to why typechecking was automatically disabled.

If someone sends us a bug report explaining why they really need this, then we can consider the added complexity.

Reply:
What I meant is for this test, not the actual `compilerOptions`.